### DISCUSSION TITLE
Adding Faust support in Astrone platform ?

### DISCUSSION THREAD

**Discussion by sletz** [Jerboas86/astrone-feedback#5] [Ideas]
Would adding [Faust](https://faust.grame.fr) support in Astrone platform makes sense ? 

Note that the Faust compiler can be [used in the Web](https://faustdoc.grame.fr/manual/deploying/).

**Comment by Jerboas86** [comment]
The new faustwasm package seems promising. i will take a look in the next month, see how it fits together.

What about Web Audio Modules, do you still support that ?

**Comment by sletz** [comment]
We'll soon add [faust2wam](https://github.com/Fr0stbyteR/faust2wam) in the [Faust IDE](https://faustide.grame.fr) remote compilation service.
